zds/tutorial/views.py

```diff
@@ -3111,14 +3116,18 @@ def get_url_images(md_text, pt):
 
                     try:
                         ext = dstroot.split(".")[-1]
-                        im = ImagePIL.open(dstroot)
-                        # if image is gif, convert to png
-                        if ext == "gif":
-                            im.save(os.path.join(dstroot.split(".")[0] + ".png"))
+                        if ext == "svg":
+                            cairosvg.svg2png(url=dstroot,
+                                             write_to=os.path.join(dstroot.split(".")[0] + ".png"))
+                        else:
+                            im = ImagePIL.open(dstroot)
+                            # if image is gif or svg, convert to png
+                            if ext == "gif":
+                                im.save(os.path.join(dstroot.split(".")[0] + ".png"))
                     except IOError:
-                        ext = dstroot.split(".")[-1]
+                        ext = dstroot.split(".")[-1].lower()
                         im = ImagePIL.open(unknow_path)
-                        if ext == "gif":
+                        if ext == "gif" or ext == "svg":
                             im.save(os.path.join(dstroot.split(".")[0] + ".png"))
                         else:
                             im.save(os.path.join(dstroot))
```

Comment:
Idem ici, la zep12 est sur le coup. Quoique si je me souviens bien, il y'a pas mal de duplications qui sont inévitable si on veut gérer tous les cas (url relative, url absolue, etc.)